util/replayer/replayer.go

```diff
@@ -15,14 +15,15 @@
 package replayer
 
 import (
+	"crypto/rand"
 	"encoding/base64"
 	"fmt"
-	"math/rand"
```

Comment:
Golang doc says `crypto/rand` is more appropriate for almost all use cases.
https://pkg.go.dev/math/rand#Read